README.md

```diff
@@ -16,6 +16,7 @@ Support for Python 2.7 has been removed, due to difficulty in installing
 - `keras`
 - `kmodes`
 - `koalas`
+- `iso3166`
```

Comment:
`i` is before `k`, reorder it before `keras`?

Reply:
fixed